data/transform/post-parser/root-level-text-wrapper.ts

```diff
@@ -0,0 +1,2 @@
+export const rootLevelTextWrapper = (content: string) =>
+  content.replace(/(<\/[A-Za-z]+>)\n([A-Z].*)\n/g, '$1\n<p>$2</p>\n');
```

Comment:
Could we please add a comment here explaining the regex? :)

Reply:
Added, explained 👍
    /**
     * This regex finds:
     * (<\/[A-Za-z]+>)\n - closing HTML tag, followed by newline.
     * ([A-Z].*)\n - a capital letter preceding any information, up to a newline.
     *
     * The overall effect is to find capitalised text that immediately follows on
     * from a closing tag at the end of a line, which will always be plain text
     * that is supposed to be wrapped in <p> elements but has been missed by the
     * textile-js parser.
     *
     * Regex101: https://regex101.com/r/j4BLqX/1
     */